mirrord/operator/src/crd/mysql_branching.rs

```diff
@@ -56,6 +56,12 @@ pub enum ConnectionSourceKind {
         container: Option<String>,
         variable: String,
     },
+
+    /// Environment from resource reference in the the pod template.
+    EnvFrom {
+        container: Option<String>,
```

Comment:
This field is never used. We should always use the container defined under target 🤷